packages/admin-ui/server/AdminUI.js

```diff
@@ -60,6 +62,102 @@ module.exports = class AdminUI {
     );
   }
 
+  staticBuild({ outputPath, apiPath, graphiqlPath }) {
+    const adminMeta = this.getAdminUIMeta({ apiPath, graphiqlPath });
+
+    const compilers = [];
+
+    const secureCompiler = webpack(
+      getWebpackConfig({
+        adminMeta,
+        entry: 'index',
+        outputPath: path.join(outputPath, 'secure'),
+      })
+    );
+    compilers.push(secureCompiler);
+
+    if (this.authStrategy) {
+      const publicCompiler = webpack(
+        getWebpackConfig({
+          // override lists so that schema and field views are excluded
+          adminMeta: { ...adminMeta, lists: {} },
+          entry: 'public',
+          outputPath: path.join(outputPath, 'public'),
+        })
+      );
+      compilers.push(publicCompiler);
+    }
+
+    return Promise.all(
+      compilers.map(
+        compiler =>
+          new Promise((resolve, reject) => {
+            compiler.run(err => {
+              if (err) {
+                reject(err);
+              } else {
+                resolve();
+              }
+            });
+          })
+      )
+    );
+  }
+
+  getAdminUIMeta({ apiPath, graphiqlPath }) {
+    const { adminPath } = this;
+
+    return {
+      adminPath,
+      apiPath,
+      graphiqlPath,
+      ...this.getAdminMeta(),
+      ...this.keystone.getAdminMeta(),
+    };
+  }
+
+  createProdMiddleware() {
+    const app = express();
```

Comment:
Should probably be an express `Router`:

```suggestion
    const app = express.Router();
```